assembly-ide/src/assembly/tomcat/bin/setenv.sh

```diff
@@ -27,9 +28,6 @@
 [ -z "${CLASSPATH}" ]  && CLASSPATH="${CATALINA_HOME}/conf/:${JAVA_HOME}/lib/tools.jar"
 
 
-export JAVA_OPTS="$JAVA_OPTS  -Dche.logs.dir=${CHE_LOGS_DIR}"
-
-
-#Class path
-[ -z "${SERVER_PORT}" ]  && SERVER_PORT=8080
-export SERVER_PORT
\ No newline at end of file
+#SERVER_PORT
```

Comment:
Why you commented these lines? Without them oauth won't work and Che won't work itself.